Assets/Scripts/Global/RemoteConfig.cs

```diff
@@ -0,0 +1,38 @@
+using System.Threading.Tasks;
+using Unity.Services.Authentication;
+using Unity.Services.Core;
+using Unity.Services.RemoteConfig;
+using UnityEngine;
+
+public class RemoteConfig : MonoBehaviour
+{
+    public struct userAttributes { }
+
+    public struct appAttributes
+    {
+        public string LatestVersion;
+        public int LatestVersionInt;
+        public string UpdateLink;
+    }
+
+    async Task InitializeRemoteConfigAsync()
+    {
+        // initialize handlers for unity game services
+        await UnityServices.InitializeAsync();
+
+        // remote config requires authentication for managing environment information
+        if (!AuthenticationService.Instance.IsSignedIn)
```

Comment:
I was going to say if we're signing in anonymously, do we even need to worry about whether we're already signed in, but I'm guessing if we are signed in we cut out an unnecessary network call here?

Reply:
My understanding (this piece was new to me) was that it required anonymous authentication which wasn't necessarily automatic--seems odd, but that's how it is. :D